cloudcannon.config.yaml

```diff
@@ -158,4 +158,31 @@ _editables:
     undo: false
     redo: false
     removeformat: false
-    copyformatting: false
\ No newline at end of file
+    copyformatting: false
+    snippet: true
+
+_snippets:
+  datawrapper:
+    template: hugo_shortcode_named_args
+    inline: false
+    preview:
+      text: Datawrapper
+    definitions:
+      shortcode_name: datawrapper
+      named_args:
+        - editor_key: title
+          type: string
+        - editor_key: src
+          type: string
+  vega:
+    template: hugo_shortcode_named_args
+    inline: false
+    preview:
+      text: vega
+    definitions:
+      shortcode_name: vega
+      named_args:
+        - editor_key: id
+          type: string
+        - editor_key: spec
+          type: string
```

Comment:
these two entries add support for the custom Vega and data wrapper shortcodes. I did not add the raw HTML shortcode because I thought the editing experience was poor. You'd be better off editing in source mode.

The two links I shared in the PR will some background on these config setting.s